test/integrations/validation/_fixtures/Zod.fixtures.ts

```diff
@@ -0,0 +1,207 @@
+
+import { InvalidObject } from '../../../../src/integrations/validation/definitions/errors.ts';
+import { ValidationSchema } from '../../../../src/integrations/validation/definitions/types';
+
+const invalidName = 'Invalid name';
```

Comment:
We typically use capital letters for these definitions: `INVALID_NAME`
Also, can we refactor these values to non domain related ones? For example: `INVALID_STRING`